digester/main.go

```diff
@@ -41,4 +147,113 @@ func main() {
 	err = db.Ping()
 	CheckError(err)
 	log.Println("Digester is now successfully connected to the tummy!")
+
+	err = unzipSource(ZIPFILE_PATH, EXTRACTION_DIR)
+	CheckError(err)
+	log.Println("Slack export has been successfully extracted!")
+
+	usersFile, err := os.ReadFile(EXTRACTION_DIR + string(os.PathSeparator) + USERS_FILENAME)
```

Comment:
I believe `filepath.Join` function can be used here instead of using `os.PathSeparator` manually.

Reply:
did something else